resources/knative-eventing/charts/knative-eventing/templates/pre-upgrade-job.yaml

```diff
@@ -57,6 +56,7 @@ data:
       subscriptions.eventing.knative.dev \
       clusterchannelprovisioners.eventing.knative.dev \
       channels.eventing.knative.dev --ignore-not-found
+    kubectl scale -n kyma-system deploy/event-bus-subscription-controller --replicas=0 || true
```

Comment:
i know this is not part of this pr and also not its purpose but:
why would this fail:
- no event-bus-subscription-controller is available => we dont need to scale
- for whatever reason scaling fails (missing permissions ... ) => we definitely want to fail
- other internal issues => we want to fail

Reply:
@k15r => from 1.6 => master upgrade scenario we need the presence of our kyma subscription controller so that once knative subscriptions are deleted during thr pre upgrade process, we immediately reconcile them, once the reconciliation is don we scale the kyma subscription to 0. @sayanh please correct me If I'm wrong or add any more details.

Reply:
@anishj0shi That's correct. We all believe `|| true` is not a good practice. Hence, changed `scale` to `delete`.